dashboard/src/components/ConfigLoader/ConfigLoader.tsx

```diff
@@ -7,6 +7,7 @@ import { ILoadingWrapperProps } from "../LoadingWrapper/LoadingWrapper";
 interface IConfigLoaderProps extends ILoadingWrapperProps {
   getConfig: () => void;
   error?: Error;
+  children?: any;
```

Comment:
`any`? Guessing this can be specific too?

Reply:
well, in this case there are too many types:
```
'string | (string & {}) | (string & ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<...>)>) | (string & ReactNodeArray) | (string & ReactPortal) | undefined'
```